functions/Errors/Error.fmfn

```diff
@@ -13,11 +13,12 @@
  *			human readable and decipherable.
  *		details = (object) JSON object including desired details
```

Comment:
I'm not sure if this pull request is the best time to bring this up, but I'll start the conversation here and maybe we can move it elsewhere if others think that's more fitting.

Two important aspects of collecting error information for me have been:

1. Being able to specify a text description of the error, like:
    - "id_Contact parameter is required"
    - "wrong context after going to layout"
    - "Not in Find mode after going to layout."
2. Being able to add additional name/value pairs to the error to help add context (like the current value of a variable).

This function seems to combine both of those into a single `details` parameter which means you only get one or the other, but can not have both, which seems wrong to me.

Reply:
I believe you could put your text description in `description` and your name/value pairs as a json object in `details`